app/src/main/java/zafrani/com/pubgapp/adapters/ItemListRecyclerViewAdapter.java

```diff
@@ -0,0 +1,90 @@
+package zafrani.com.pubgapp.adapters;
+
+import android.content.Context;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import java.util.List;
+
+import zafrani.com.pubgapp.R;
+
+public class ItemListRecyclerViewAdapter extends RecyclerView.Adapter<RecyclerView.ViewHolder> {
+    private List<Object> feedItemList;
+    private Context mContext;
+
+    public ItemListRecyclerViewAdapter(Context context, List<Object> feedItemList) {
+        this.feedItemList = feedItemList;
+        this.mContext = context;
+    }
+
+    @Override
+    public RecyclerView.ViewHolder onCreateViewHolder(ViewGroup viewGroup, int i) {
```

Comment:
btw if you press **alt** + **enter** -> **Implement methods**, Android Studio should populate the methods for you with the right naming